nvidia_gpu_stats/cmd/nvidia_gpu_stats/main.go

```diff
@@ -0,0 +1,241 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/shirou/gopsutil/v4/process"
+
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+)
+
+type GPUNvidia struct {
```

Comment:
doc should say what this represents. a single GPU? what's nvmlInit, and if it's not initialized in `NewGPUNvidia`, then when is it mutated and what guarantees are there?